### PR TITLE
fix previous convention pull 3593

### PR DESCRIPTION
**Pull request by dinhquang251**
fix previous convention pull 3593

### CODE REVIEW

**Comment by dinhquang251** [comment]
ready

File: app/src/main/java/com/framgia/quangtran/music_42/data/source/genre/GenreLocalDataSource.java [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+                GenreKey.ALL_AUDIO, R.drawable.alldaudio));
+        genres.add(new Genre(GenreName.ALTERNATIVE,
+                GenreKey.ALTERNATIVE, R.drawable.alternative));
+        genres.add(new Genre(GenreName.AMBIENT,
+                GenreKey.AMBIENT, R.drawable.ambient));
+        genres.add(new Genre(GenreName.CLASSICAL,
+                GenreKey.CLASSICAL, R.drawable.classical));
+        genres.add(new Genre(GenreName.COUNTRY,
+                GenreKey.COUNTRY, R.drawable.country));
+        if (genres != null) {
```
Comment (DoanVanToan) [inline app/src/main/java/com/framgia/quangtran/music_42/data/source/genre/GenreLocalDataSource.java]: biết chắc chắn genre k thể null được rồi
em k cần check nữa

File: app/src/main/java/com/framgia/quangtran/music_42/data/repository/GenreRepository.java [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+public class GenreRepository implements GenreDataSource.LocalGenre {
+    private static GenreRepository sInstance;
+    private GenreDataSource.LocalGenre mDataSource;
+
+    private GenreRepository(GenreDataSource.LocalGenre localGenre) {
+        mDataSource = localGenre;
+    }
+
+    public static synchronized GenreRepository
+    getInstance(GenreDataSource.LocalGenre localGenre) {
```
Comment (NguyenMinhTuanB) [inline app/src/main/java/com/framgia/quangtran/music_42/data/repository/GenreRepository.java]: Enter tùy ý quá em

**Comment by NguyenMinhTuanB** [comment]
commented

**Comment by dinhquang251** [comment]
ready

**Review by NguyenMinhTuanB — APPROVED** [review]

**Pull request merged**